src/ui/menu.c

```diff
@@ -74,12 +74,12 @@ void ui_menu_main() {
         ux_stack_push();
     }
 
-    if (G_coin_config->kind == COIN_KIND_BITCOIN) {
+    if (G_coin_config->bip32_pubkey_version == 0x0488B21E) {  // mainnet
         ux_flow_init(0, ux_menu_main_flow_bitcoin, NULL);
-    } else if (G_coin_config->kind == COIN_KIND_BITCOIN_TESTNET) {
+    } else if (G_coin_config->bip32_pubkey_version == 0x0488B21E) {  // testnet
```

Comment:
can you enter here? looks like the same as the if above

Reply:
isnt the testnet 0x043587CF ? also use defines for that please !

Reply:
Oops, good catch. Fixed in 130a71bec (and also updated the :copyright: notice, while I was there)

Reply:
Added tests to catch such bloopers in 7d0f0cfa